c++/src/capnp/compat/http-over-capnp.c++

```diff
@@ -852,9 +852,6 @@ public:
   kj::Promise<void> request(RequestContext context) override {
     // Common implementation of request() and startRequest(). callback() performs the
```

Comment:
I think I deleted all the code for `startRequest`, so perhaps this part of the comment is obsolete too?

Reply:
Yeah, I think the whole comment is obsolete since startRequest() and the callback no longer exist, I dropped it.